packages/app/src/app/overmind/effects/vscode/index.ts

```diff
@@ -216,7 +216,11 @@ export class VSCodeEffect {
     moduleShortid: string,
     operation: (string | number)[]
   ) {
-    return this.modelsHandler.applyOperation(moduleShortid, operation);
+    if (!this.modelsHandler) {
```

Comment:
Poh, so the operation gets dropped? I mean, I understand it because we sync state, maybe we need to put a comment here. I can imagine someone seeing this as a bug 😂